### DISCUSSION TITLE
utility meter a orario

### DISCUSSION THREAD

**Discussion by kimuss** [indomus/forum#1546] [HUB: Home Assistant]
salve,
vorrei creare un sensore che mi dica quanta energia (kwh) consumo dalle 11:00 alle 17:00 di ogni giorno.
al momento ho un sensor integration che mi calcola i kwh da un sensore che legge il consumo, però ovviamente è un dato che mi ristituisce il totale. Con la parte "energy"  nella sidebar ho visto che posso dividerlo per giorni, ma non mi sembra di aver trovato dei filtri per vedere solo alcune ore del giorno.

mi consigliate di fare un utility meter che punta a quel sensor integration, daily con offset?
per avere una media che si prolunga per più di una settimana (tempo che ha il recorder per cancellare i dati) cosa devo fare??
grazie delle risposte

**Comment by indomus** [comment]
Prendi spunto [da qui](https://indomus.it/progetti/configurare-dei-contabilizzatori-di-consumo-energetico-su-home-assistant-v2/).
